corehq/apps/hqadmin/templates/hqadmin/raw_couch.html

```diff
@@ -7,10 +7,13 @@
         <link type="text/css" rel="stylesheet" media="all" href="{% static 'style/lib/bootstrap-3.2.0/dist/css/bootstrap.min.css' %}"/>
         <link rel="stylesheet" href="{% static 'codemirror/lib/codemirror.css' %}" />
         <link rel="stylesheet" href="{% static 'codemirror/addon/fold/foldgutter.css' %}"/>
+        <link type="text/css" rel="stylesheet" media="all" href="{% new_static 'style/lib/select2/select2.css' %}" />
```

Comment:
good point. Looks like this version is 3.5.2, the version we have in bower is 3.4.5 ("select2-3.4.5-legacy"), and also in /style/ we have a 4.0...,